pom.xml

```diff
@@ -114,48 +114,12 @@
             <groupId>org.apache.hive</groupId>
             <artifactId>hive-exec</artifactId>
             <version>${hive.version}</version>
-            <exclusions>
```

Comment:
Have you tried running all tests with tez and mr?

Also have you checked it works for some dummy Java project with HiveRunner as test dependency - I know I had a lot of problems with that, thats why I have excluded all of it.

Reply:
I just ran `mvn package` locally. If there are other variations I should try let me know. 

I haven't tried it on a dummy project, I no longer work for the company where we used all of this and where I used to have a big test suite. We should create a dummy project on GitHub in this org that we can use for that.